src/accounts.rs

```diff
@@ -167,12 +167,17 @@ impl Accounts {
             .with_context(|| format!("no account with id {id}"))?;
         ctx.stop_io().await;
 
-        // Explicitly close the database.
+        // Explicitly close the database
+        // to make sure the database file is closed
+        // and can be removed on Windows.
+        // If some spawned task tries to use the database afterwards,
+        // it will fail.
         //
-        // Stopping I/O aborts the tasks that keep `Context` clones,
-        // but aborting the task does not immediately drops the future.
-        // To make sure the database file is closed
-        // and can be removed on Windows, we drop all the connections manually.
+        // Previously `stop_io()` aborted the tasks without awaiting them
+        // and this resulted in keeping `Context` clones inside
+        // `Future`s that were not dropped. This bug has since been fixed,
```

Comment:
The commit fixes the bug, but says that it has been fixed. Maybe "is fixed now" is better, but not very important

Reply:
Yes, but when i read this comment i feel that this one fix is mentioned, not the previous which closes the db in remove_account()